tests/hpc/rasdaemon.pm

```diff
@@ -47,8 +47,13 @@ sub run {
 
     assert_script_run('! ras-mc-ctl --status');
 
+    # Try to start rasdaemon. May need a restart on aarch64.
     systemctl('start rasdaemon');
-    systemctl('is-active rasdaemon');
+    if (systemctl('is-active rasdaemon', ignore_failure => 1)) {
+        systemctl('restart rasdaemon');
+        record_info('RESTART', "rasdaemon service was inactive and was restarted.");
```

Comment:
Maybe mark this a a soft failure, since this is a known deviation from a regular green run.